web/Views/Shared/_Layout.cshtml

```diff
@@ -26,18 +26,27 @@
                         <li class="nav-item">
                             <a class="nav-link text-dark" asp-area="" asp-controller="Home" asp-action="Privacy">Privacy</a>
                         </li>
-                        <li class="nav-item" style="margin-left: auto;">
-                            @{
-                                if (User.Identity?.IsAuthenticated == true)
-                                {
-                                    <a href="/TODO/profile" class="nav-link">@(((BricksAndHeartsUser)User.Identity).GoogleEmail)</a>
-                                }
-                                else
+                    </ul>
+                    
+                    <ul class="navbar-nav ms-auto">
+                        @if (User.Identity?.IsAuthenticated == true)
+                        {
+                            <li class="nav-item" style="margin-left: auto;">
+                                <a href="/TODO/profile" class="nav-link">@(((BricksAndHeartsUser)User.Identity).GoogleEmail)</a>
+                            </li>

```

Comment:
Feel free to remove this, we can always add it back with a real link sometime

Reply:
ok we've removed them.

Reply:
We've removed the `href` for now :)